src/markdown-pages/automate-workflows/get-started-new-relic-cli.mdx

```diff
@@ -78,24 +78,26 @@ newrelic profiles default -n tutorial
 
 </Step><Step>
 
-## Get your application details
+Get your application details.
 
-In this example, you are going to add tags to the application you've instrumented with New Relic. [Tags](https://docs.newrelic.com/docs/new-relic-one/use-new-relic-one/core-concepts/tagging-use-tags-organize-group-what-you-monitor) are key-value pairs that can help you organize and filter your entities. An [entity](https://docs.newrelic.com/docs/new-relic-one/use-new-relic-one/core-concepts/what-entity-new-relic) (for example, an application) can have a maximum of 100 key-value pairs tied to it.
+Now, add tags to the application you've instrumented with New Relic. [Tags](https://docs.newrelic.com/docs/new-relic-one/use-new-relic-one/core-concepts/tagging-use-tags-organize-group-what-you-monitor) are key-value pairs that can help you organize and filter your entities. An [entity](https://docs.newrelic.com/docs/new-relic-one/use-new-relic-one/core-concepts/what-entity-new-relic) (for example, an application) can have a maximum of 100 key-value pairs tied to it.
 
 Before searching for your application using the New Relic CLI, write down or copy your [Account ID](https://docs.newrelic.com/docs/accounts/install-new-relic/account-setup/account-id) and the [name of your application in New Relic](https://docs.newrelic.com/docs/agents/manage-apm-agents/app-naming/name-your-application) - you need both to find applications in the New Relic platform.
 
 </Step><Step>
 
-The New Relic CLI can retrieve your application details as a JSON object.
+Retrieve your application details as a JSON object.
 
-To search for your APM application use the [`apm application search`](https://github.com/newrelic/newrelic-cli/blob/master/docs/cli/newrelic_apm_application_search.md) command. If you get an error, check that the account ID and application name you provided are correct.
+To search for your APM application, use the [`apm application search`](https://github.com/newrelic/newrelic-cli/blob/master/docs/cli/newrelic_apm_application_search.md) command. If you get an error, check that the account ID and application name you provided are correct.
```

Comment:
I agree with you on a lot of the consistency issues you've raised, but this one doesn't bother me for a couple of reasons. One is, if you're following the steps as presented, you don't need to be reminded that it's a command. And the second is, in the effort to eradicate robot voice, I'm for a little variation in places where it won't be too obvious. We could also achieve that by not labeling it a command.

Reply:
Ok, that makes sense! FWIW, I am actually in favor of always referring to it in the “object” form, not as an adjective because it feels less robotic.

So, in this case, I like:

> ... use `apm application search`.

Better than:

> ... use the `apm application search` command.

But point taken on liking variation.